stablehlo/reference/Element.cpp

```diff
@@ -502,6 +502,14 @@ Element log(const Element &el) {
       [](std::complex<double> e) { return std::log(e); });
 }
 
+Element logistic(const Element &el) {
+  Element one;
```

Comment:
Given that 

(1) `Element` equivalents of ones/zeros are something we might need in  other places as well
(2) For creating an `Element` obj out of a arbitrary real, imag pair, we use 
```
  Element(Type type, std::complex<double> value);
```
This constructor is mostly used with creating `Element` equivalent of a complex value with imag part == 0. For e.g its usage in the `iota`. IMO, this function will be rarely used to get Element equiv of an arbitrary complex values with imag != 0. 

With that, how about we  use 
```
  Element(Type type, double value);
```
for creating both complex and float types, with proper comments explaining the API.

With that we can get:
```
Element one(el.getType(), 1.0);
return one / (one + exponential(-el));
```

Additionally,
We can chose to remove the following:
```

Element(Type type, std::complex<double> value); // no compelling use
Element(Type type, bool); // Removing it should not affect any usages as  
                                            // Element(Type type, int64_t) will be used 
                                             // as alternative 
```

Given that 
The logistic function turns out to be trivially implementable (by which I meant without any explicit code to handle type differences), we can also chose to inline it in the Ops.cpp.

Reply:
Thanks for the suggestion. I agree with most of your suggestion but let's keep the implementation in Ops.cpp to be consistent with how we provide implementations for LogOp in the same file.